gtsfm/utils/metrics.py

```diff
@@ -289,7 +290,9 @@ def compute_ba_pose_metrics(
         A group of metrics that describe errors associated with a bundle adjustment result (w.r.t. GT).
     """
     wTi_aligned_list = ba_output.get_camera_poses()
-    i2Ui1_dict_gt = get_twoview_translation_directions(gt_wTi_list)
+    possible_img_pair_idxs = list(itertools.combinations(range(len(wTi_aligned_list)), 2))
+
+    i2Ui1_dict_gt = get_twoview_translation_directions(gt_wTi_list, possible_img_pair_idxs)
```

Comment:
already added another comment about this change.

Reply:
Addressed it. Please recheck.